packages/flutter/test/widgets/heroes_test.dart

```diff
@@ -1448,4 +1448,40 @@ void main() {
     ));
     expect(find.text('two'), findsOneWidget);
   });
+
+  testWidgets('Hero within a Hero, throws', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Material(
+          child: Hero(
+            tag: 'a',
+            child: Hero(
+              tag: 'b',
+              child: Text('Child of a Hero'),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    expect(tester.takeException(), isAssertionError);
+  });
+
+  testWidgets('Hero within a Hero subtree, throws', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Material(
+          child: Hero(
```

Comment:
Isn't this the same as the previous test?

Reply:
Yes, I somehow changed it while ... formatting it ... that must be a first 🙃